JS/oop-assignment-forum/User.js

```diff
@@ -0,0 +1,12 @@
+export default class User {
+  constructor({ userName }) {
+    this.userName = userName;
+  }
+  writePost(post, page) {
+    page.createPost(post)
+  }
+  writeComment(comment, post) {
+    post.createComment(comment)
+    post.comments[comment.id] = comment;
```

Comment:
`post.comments[comment.id] = comment;` not a fan of this - why isn't this done on the Post class?